### PR TITLE
[Translation] [LocoProvider] Add content-type for POST translations

### LINKED ISSUE #44031: [loco-translation-provider] Improper content type of http request
### Symfony version(s) affected

5.3

### Description

When using Loco provider is impossible to send single word "data" as a translation.

```POST /translations/tpl.patients.data/pl HTTP/1.1
Host: localize.biz
Authorization: ****
Accept: */*
User-Agent: Symfony HttpClient/Curl
Accept-Encoding: gzip
Content-Length: 4
Content-Type: application/x-www-form-urlencoded

data
```
When set `Content-Type: text/plain` it works.

### How to reproduce

Tested with symfony/http-client v5.3.10

Push source asset translation: data

### Possible Solution

Workaround:
new CurlHttpClient(['headers' => ['Content-type' => 'text/plain']])

Solution:
Force http client  to use 'Content-type' => 'text/plain'

### Additional Context

_No response_

### PR DESCRIPTION
**Pull request by tomasz-kusy**
| Q             | A
| ------------- | ---
| Branch?       | 5.4
| Bug fix?      | yes
| New feature?  |no
| Deprecations? | no
| Tickets       | Fix #44031
| License       | MIT
https://github.com/symfony/symfony/issues/44031

### CODE REVIEW

**Review by Kocal — APPROVED** [review]

File: src/Symfony/Component/Translation/Bridge/Loco/LocoProvider.php [hunk 1]
```diff
@@ -207,6 +207,7 @@ private function translateAssets(array $translations, string $locale): void
         foreach ($translations as $id => $message) {
             $responses[$id] = $this->client->request('POST', sprintf('translations/%s/%s', rawurlencode($id), rawurlencode($locale)), [
                 'body' => $message,
+                'headers' => ['Content-Type' => 'text/plain'],
```
Comment (nicolas-grekas) [inline src/Symfony/Component/Translation/Bridge/Loco/LocoProvider.php:210]: This line should be added to all calls that have a string as body.

**Review by nicolas-grekas — COMMENTED** [review]
Thanks for the PR. Just one comment and good to go on my side.

File: src/Symfony/Component/Translation/Bridge/Loco/LocoProvider.php (hunk 1 above)
Comment (tomasz-kusy) [reply src/Symfony/Component/Translation/Bridge/Loco/LocoProvider.php:210]: I've changed only where I have tested.
Another place where you suggest to add is POST route to 'tags/%s.json'.
But I'm not sure if there 'text/plain' is suitable.
The documentation suggests 'application/json': https://localise.biz/api#!/tags/tagAssets
obraz

Comment (nicolas-grekas) [reply src/Symfony/Component/Translation/Bridge/Loco/LocoProvider.php:210]: We should review the calls and put the content-type that makes sense. Sending application/json only if we send actual json, etc. Right now we never do so. It'd be great confirming that the other endpoints work well of course.

**Comment by fabpot** [comment]
Thank you @tomasz-kusy.

**Pull request merged**